UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -184,6 +184,10 @@ private void OnSceneChange(Scene oldScene, Scene newScene)
 	{
 		foreach (var sound in Instance.SoundSpawns)
 		{
+			if (sound.Value == null)
```

Comment:
leave a comment to make sure we remember later this is a hackfix, something with a //FIXME for example.